src/ansible_navigator/configuration_subsystem/navigator_configuration.py

```diff
@@ -88,6 +89,17 @@ def generate_share_directory():
     return share_directory
 
 
+def retrieve_default_ee_image() -> str:
```

Comment:
Add lru caching here, the value never changes.

Reply:
runtime it only gets called once at initialization

it will get called multiple time in the tests, but I intend to replace the CONSTANT with a session fixture eventually